tests/itest_specs/basic_round_trip/long_call.cairo

```diff
@@ -179,20 +179,20 @@ namespace LongCallRoundTrip {
         %{ stop_warp_1 = warp(1000000000 + 60*60*12, target_contract_address=ids.amm_addr) %}
 
         let strike_price = Math64x61.fromFelt(1500);
-        let one = Math64x61.fromFelt(1);
+        let one_option_size = 1 * 10**18;
 
         let (premia: Math64x61_) = IAMM.trade_open(
             contract_address=amm_addr,
             option_type=0,
             strike_price=strike_price,
             maturity=expiry,
             option_side=0,
-            option_size=one,
+            option_size=one_option_size,
             quote_token_address=myusd_addr,
             base_token_address=myeth_addr
         );
 
-        assert premia = 2020558154346487; // approx 0.0087 ETH
+        assert premia = 1391658545716339; // approx 0.0006 ETH
```

Comment:
The difference here is just bizzarre

Reply:
Did you get to find out where is the difference after the premia assertion? ie if it's in the volatility, or balance or sth

Reply:
or if there is even a difference somewhere else in the test at all

Reply:
Not yet, working on it now

Reply:
@Chepelau Now pushed with debug asserts, both change by a lot